config/locales/server.en.yml

```diff
@@ -638,7 +638,6 @@ en:
     request_membership_pm:
       title: "Membership Request for @%{group_name}"
     request_accepted_pm:
-      title: "You've been accepted into @%{group_name}"
```

Comment:
Since this is no longer a Topic, the title field here is not needed. Slashed this one for the PR, but leaving the others for now in case we don't want to take this line out.

Reply:
@pmusaraj Should I remove this line from the other ymls?

Reply:
> @pmusaraj Should I remove this line from the other ymls?

No, removing it from the English locale file is enough. The other locale files are updated automatically.

Reply:
Awesome, thank you!